graylog2-server/src/main/java/org/graylog2/periodical/VersionCheckThread.java

```diff
@@ -179,7 +179,7 @@ public boolean isDaemon() {
 
     @Override
     public int getInitialDelaySeconds() {
-        return 0;
+        return (int) MINUTES.toSeconds(5);
```

Comment:
I'd recommend using `Ints.saturatedCast(MINUTES.toSeconds(5L))` here or create a constant `private final int SECONDS_PER_MINUTE = 60;` which can be multiplied by the number of minutes (of course we also need `SECONDS_PER_HOUR`, `SECONDS_PER_DAY`, and `SECONDS_PER_LUNAR_YEAR`).